lib/ably/realtime/channel.rb

```diff
@@ -146,13 +146,13 @@ def initialize(client, name, channel_options = {})
       #   end
       #
       def publish(name, data = nil, attributes = {}, &success_block)
-        if detached? || detaching? || failed?
+        if suspended? || failed?
           error = Ably::Exceptions::ChannelInactive.new("Cannot publish messages on a channel in state #{state}")
           return Ably::Util::SafeDeferrable.new_and_fail_immediately(logger, error)
         end
 
         if !connection.can_publish_messages?
-          error = Ably::Exceptions::MessageQueueingDisabled.new("Message cannot be published. Client is configured to disallow queueing of messages and connection is currently #{connection.state}")
+          error = Ably::Exceptions::MessageQueueingDisabled.new("Message cannot be published. Client is not allowed to queue of messages when connection is in state #{connection.state}")
```

Comment:
"to queue of messages" -> "to queue messages"

Reply:
Thanks, good spot.